### ISSUE TITLE
Linked Vocabs?

### ISSUE THREAD

**Issue by terrellt** [curationexperts/oargun#1]
There seems to be a big overlap between this and https://github.com/no-reply/linked_vocabs (understandable, since they were both extracted from Oregon Digital.)

**Comment by no-reply** [comment]
Agreed.  Is there something separating this from Linked Vocabs, at this point?  Is there something about Linked Vocabs that takes this in the wrong direction?

Could we converge during the post-Code4Lib meetings?

**Comment by jcoyne** [comment]
I'll be able to take another look when I'm back on the UCSD project next week.  Originally I said "I want the models from OD", so I started this project with that in mind.  I was unaware of Linked Vocabs.  I suspect I want a bit more than Linked Vocabs is offering but also not quite what OD has either.
